backend/jobsearch/src/main/resources/application.properties

```diff
@@ -19,6 +19,6 @@ jwt.expiration=86400
 
 # create-drop for test
 # update to persist
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
Why did you change this? I guess we should not drop table

Reply:
my mistake, I didn't noticed it was pushed

Reply:
please remove it and resolve the merge conflicts and I'll approve it